assets/src/stories-editor/blocks/amp-story-page/copy-paste-handler.js

```diff
@@ -45,20 +49,26 @@ function CopyPasteHandler( { children, onCopy, onCut, clientId, isSelected } ) {
 		let plainText = '';
 		let html = '';
 
-		// IE11 only supports `Text` as an argument for `getData` and will
-		// otherwise throw an invalid argument error, so we try the standard
-		// arguments first, then fallback to `Text` if they fail.
-		try {
-			plainText = clipboardData.getData( 'text/plain' );
-			html = clipboardData.getData( 'text/html' );
-		} catch ( error1 ) {
+		if ( getCopiedMarkupState().length ) {
+			plainText = getCopiedMarkupState();
+			html = getCopiedMarkupState();
+			clearCopiedMarkup();
```

Comment:
`clear`? So you can only paste once? Is this by design?

Reply:
The same limitation existed in #3083 but was removed in ddce16429cb85a5b10ec5429d62ce88ac5866723. I don't see why cut should now behave differently than copy.

Reply:
In my testing it now behaved like this:

1. Cut block A
1. Block A is removed and block b is selected
1. Select page so that pasting works
1. Paste (expectation: copy of A is inserted)
1. Newly inserted block A\' is selected
1. Cut block B
1. Select page so that pasting works
1. Paste (expectation: copy of B is inserted)
1. Newly inserted block B\' is selected
1. Select page so that pasting works
1. Paste (expectation: copy of B is inserted)
1. Newly inserted block A\'' is selected

I suppose this is because Cmd+X adds the content to both the store and the actual clipboard, and after pasting it is cleared from the store but not the actual clipboard. That means on subsequent paste it will paste the old information.

Reply:
If you cut text normally, you can paste it as many times as you want, as it is on the clipboard. So what is the expected behavior here.

Reply:
This is out of date this call to `clearCopiedMarkup` was removed.